Assets/BossRoom/Scripts/Shared/Net/ConnectionManagement/GameNetPortal.cs

```diff
@@ -87,6 +88,8 @@ public class GameNetPortal : MonoBehaviour
         private LocalLobby m_LocalLobby;
         private LobbyServiceFacade m_LobbyServiceFacade;
 
+        public OnlineMode OnlineMode { get; set; }
```

Comment:
Same here, we should only have one source of truth

Reply:
I removed this field entirely, I don't really think it is required here, the GameNetPortal can simply return auth's playerId or the guid based on if we are connected or not.